golang/.gitignore

```diff
@@ -3,6 +3,4 @@
 .DS_Store
 build
 Godeps/_workspace
-messagetype_string.go
-reqresreaderstate_string.go
-reqreswriterstate_string.go
+*_string.go
```

Comment:
I have a slight preference for not checking in generated code, but that's mostly a holdover from the old days of terrible IDL.  Whatever you prefer.